content/docs/esc/administration/self-hosting.md

```diff
@@ -0,0 +1,21 @@
+---
+title_tag: "Self-hosting the Pulumi ESC"
```

Comment:
This is missing a subject for the article "the". Could be "Self-hosting Pulumi ESC" or one of "Self-hosting the Pulumi ESC {service, backend, api}"